### PR TITLE
update iv_msgs -> auto_msgs in planning readme

### PR DESCRIPTION
**Pull request by takayuki5168**
## Description

In planning README.md, I updated iv_msgs -> auto_msgs

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by takayuki5168** [comment]
@taikitanaka3 I forgot this one when dealing with planning readme. Now it's merge ready

File: planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/surround_obstacle_checker-design.ja.md [hunk 1]
```diff
@@ -46,20 +46,20 @@ Stop condition の項で述べたように、状態によって障害物判定
...
-| `/localization/twist`                    | `geometry_msgs::msg::TwistStamped`                  | Current twist        |
-| `/tf`                                    | `tf2_msgs::msg::TFMessage`                          | TF                   |
-| `/tf_static`                             | `tf2_msgs::msg::TFMessage`                          | TF static            |
+| Name                                     | Type                                                     | Description          |
+| ---------------------------------------- | -------------------------------------------------------- | -------------------- |
+| `~/input/trajectory`                     | `autoware_auto_planning_msgs::msg::Trajectory`           | Reference trajectory |
+| `/sensing/lidar/no_ground/pointcloud`    | `sensor_msgs::msg::PointCloud2`                          | No ground pointcloud |
+| `/perception/object_recognition/objects` | `autoware_auto_perception_msgs::msg::DynamicObjectArray` | Dynamic objects      |
+| `/localization/twist`                    | `geometry_msgs::msg::TwistStamped`                       | Current twist        |
```
Comment (taikitanaka3) [inline planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/surround_obstacle_checker-design.ja.md:54]: `localization/twist?`

File: planning/scenario_planning/common/motion_velocity_smoother/README.md [hunk 2]
```diff
@@ -86,13 +86,13 @@ After the optimization, a resampling called `post resampling` is performed befor
...
-| `/planning/scenario_planning/max_velocity` | `std_msgs/Float32`                  | External velocity limit [m/s] |
-| `/localization/odometry`                   | `nav_msgs/Odometry`                 | Current odometry              |
-| `/tf`                                      | `tf2_msgs/TFMessage`                | TF                            |
-| `/tf_static`                               | `tf2_msgs/TFMessage`                | TF static                     |
+| Name                                       | Type                                     | Description                   |
+| ------------------------------------------ | ---------------------------------------- | ----------------------------- |
+| `~/input/trajectory`                       | `autoware_auto_planning_msgs/Trajectory` | Reference trajectory          |
+| `/planning/scenario_planning/max_velocity` | `std_msgs/Float32`                       | External velocity limit [m/s] |
+| `/localization/odometry`                   | `nav_msgs/Odometry`                      | Current odometry              |
```
Comment (taikitanaka3) [inline planning/scenario_planning/common/motion_velocity_smoother/README.md:93]: want to modify here too `localization/kinematic_state` code modify PR can be later

File: planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/surround_obstacle_checker-design.ja.md (hunk 1 above)
Comment (takayuki5168) [reply planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/surround_obstacle_checker-design.ja.md:54]: Ah, this file was supposed to be removed, but it was decided no to be removed. So I'll fix. Thanks

File: planning/scenario_planning/common/motion_velocity_smoother/README.md (hunk 2 above)
Comment (takayuki5168) [reply planning/scenario_planning/common/motion_velocity_smoother/README.md:93]: Oh I couldn't find that, thanks!

File: planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/surround_obstacle_checker-design.ja.md (hunk 1 above)
Comment (takayuki5168) [reply planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/surround_obstacle_checker-design.ja.md]: https://github.com/tier4/AutowareArchitectureProposal.iv/pull/576/commits/cfb7d2d87ec9f4479f6ee74422af82100868a3dd
Done

File: planning/scenario_planning/common/motion_velocity_smoother/README.md (hunk 2 above)
Comment (takayuki5168) [reply planning/scenario_planning/common/motion_velocity_smoother/README.md]: https://github.com/tier4/AutowareArchitectureProposal.iv/pull/576/commits/cfb7d2d87ec9f4479f6ee74422af82100868a3dd
DONE

Comment (takayuki5168) [reply planning/scenario_planning/common/motion_velocity_smoother/README.md]: https://github.com/tier4/AutowareArchitectureProposal.iv/pull/576/commits/3167bf631f3a79661cacfb569d5ea0cdf98feda3

**Review by taikitanaka3 — APPROVED** [review]
LGTM

**Pull request merged**